oxidation/libparsec/crates/platform-async/src/platform/default.rs

```diff
@@ -20,7 +20,7 @@ where
     let task = Task::new(shared_state.clone());
     let runnable = Runnable::new(future, shared_state);
 
-    let _handle = tokio::spawn(runnable);
+    drop(tokio::spawn(runnable));
```

Comment:
why drop ?
It reaches out of scope after the return

Reply:
I don't need the handle returned by tokio.

I've removed the drop in 847d250